hpycc/get.py

```diff
@@ -321,66 +326,56 @@ def get_thor_file(connection, thor_file, max_workers=15, chunk_size=10000,
     >>> df = pandas.DataFrame({"col1": [1, 2, 3]})
     >>> df.to_csv("example.csv", index=False)
     >>> hpycc.spray_file(conn,"example.csv","example")
-    >>> hpycc.get_logical_file(conn, "example", dtype=str)
+    >>> hpycc.get_thor_file(conn, "example", dtype=str)
         col1
     0     '1'
     1     '2'
     2     '3'
 
     """
-    url = ("http://{}:{}/WsWorkunits/WUResult.json?LogicalName={}"
-           "&Cluster=thor&Start={}&Count={}").format(
-        connection.server, connection.port, thor_file, 0, 1)
-    r = connection.run_url_request(url, max_attempts, max_sleep)
-    rj = r.json()
-    try:
-        wuresultresponse = rj["WUResultResponse"]
-    except KeyError:
-        raise KeyError("json: {}".format(rj))
-    schema_str = wuresultresponse["Result"]["XmlSchema"]["xml"]
-
-    # get the schema as named tuples of (name, is_set, type)
-    schema = parse_schema_from_xml(schema_str)
 
-    num_rows = wuresultresponse["Total"]
-
-    # if there are no rows to go and get, we should return an empty dataframe
-    if not num_rows:
-        cols = [c.name for c in schema]
-        return pd.DataFrame(columns=cols)
+    resp = connection.get_chunk_from_hpcc(thor_file, 0, 1, max_attempts, max_sleep, min_sleep)
+    try:
+        wuresultresponse = resp["WUResultResponse"]
+        schema_str = wuresultresponse["Result"]["XmlSchema"]["xml"]
+        schema = parse_schema_from_xml(schema_str, dtype)
```

Comment:
I think this dtype bit shouldn't be dealt with here, and should just be overridden if manual. It keeps the parse schema function cleaner